generation/WinSDK/enums.json

```diff
@@ -30,6 +30,75 @@
       {
         "method": "GetThemeSysColorBrush",
         "parameter": "iColorId"
+      },
+      {
+        "method": "GetThemeColor",
+        "parameter": "iPropId"
+      },
+      {
+        "method": "GetThemeSysBool",
+        "parameter": "iBoolId"
+      },
+      {
+        "method": "GetThemeSysInt",
+        "parameter": "iIntId"
+      },
+      {
+        "method": "GetThemeInt",
+        "parameter": "iPropId"
+      },
+      {
+        "method": "GetThemeIntList",
+        "parameter": "iPropId"
+      },
+      {
+        "method": "GetThemePosition",
+        "parameter": "iPropId"
+      },
+      {
+        "method": "GetThemeMargins",
+        "parameter": "iPropId"
+      },
+      {
+        "method": "GetThemeFilename",
+        "parameter": "iPropId"
+      },
+      {
+        "method": "GetThemeEnumValue",
+        "parameter": "iPropId"
+      }
+    ]
+  },
+  {
+    "name": "SET_THEME_APP_PROPERTIES",
```

Comment:
Hm, slightly confusing to have a *_PROPERTIES suffix on a set of flags to pass into `dwFlags`. Perhaps `SET_THEME_APP_PROPERTIES_FLAGS` instead?

Reply:
done.